app/bundles/ReportBundle/Tests/Functional/AbstractReportSubscriberTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
```

Comment:
Everything is fine, the changes are showing up for me, but the unittests are not passing because PHPUnit tries to run tests on the abstract class AbstractReportSubscriberTest, which is not possible. This results in a warning during the test run.

Reply:
Thanks! I've renamed the `AbstractReportSubscriberTest` to `AbstractReportSubscriberTestCase` so it won't qualify as test.